src/data/ua_overrides.js

```diff
@@ -1264,6 +1264,23 @@ const AVAILABLE_UA_OVERRIDES = [
       },
     },
   },
+  {
+    /*
+     * Bug 1836116 - UA override for www.slushy.com
+     *
+     * The site's content is not loaded without a Chrome UA spoof.
+     */
+    id: "bug1836116",
+    platform: "all",
+    domain: "www.slushy.com",
+    bug: "1836116",
+    config: {
+      matches: ["*://www.slushy.com/*"],
+      uaTransformer: originalUA => {
+        return originalUA + " Chrome/15.0.0.0";
```

Comment:
Is version 15 a typo or deliberate? Asking only because this is a version from 2011. :)

Reply:
It's just the first version I tried. And it confirms that we don't have to worry about keeping the version number up-to-date, so I left it at that. I can try with 0.0.0.0, or even use the frozen version number Chrome has decided to use instead?

Reply:
Chrome's frozen number would probably be the best solution (and it would mean we don't have to worry).